release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -160,6 +160,12 @@ These new options include host configuration, network configuration, and baseboa
 These fields are not used during the initial provisioning of the cluster, but they eliminate the need to set the fields after installation.
 For more information, see xref:../installing/installing_with_agent_based_installer/installation-config-parameters-agent.adoc#installation-configuration-parameters-additional-bare_installation-config-parameters-agent[Additional bare metal configuration parameters for the Agent-based Installer].
 
+[id="ocp-4-15-baremetal-raid-dell"]
+==== Use the Dell iDRAC to configure a RAID during installer-provisioned installation
+
+You can now use the Dell iDRAC with Redfish to configure a redundant array of independent disks (RAID) for the bare metal platform by using baseboard management controllers (BMCs) during an installer-provisioned installation.
```

Comment:
You can now use the Dell iDRAC BMC with the Redfish protocol to configure a Redundant Array of Independent Disks (RAID) for the bare metal platform during an installer-provisioned installation.

Reply:
Thanks! I will update the PR.